src/sonic-yang-models/yang-templates/sonic-policer.yang.j2

```diff
@@ -0,0 +1,126 @@
+{% if yang_model_type == "cvl" %}
+/* this is sonic cvl yang model */
+{% else %}
+/* this is sonic py yang model */
+{% endif %}
```

Comment:
What is the purpose to be a jinja template and have these comments?

Reply:
The j2 template was introduced recently. The main purpose is to support both config management class which DPB and other sonic applications use as well as sonic-mgmt framework.
Currently this is just used for comments but in the future we could add more j2 checks. So It is better to define it as j2 instead of json.